contrib/local-environment/docker-compose.yaml

```diff
@@ -13,7 +13,7 @@ version: '3.0'
 services:
   oauth2-proxy:
     container_name: oauth2-proxy
-    image: quay.io/oauth2-proxy/oauth2-proxy:v7.2.1
+    image: quay.io/oauth2-proxy/oauth2-proxy:${VERSION}
```

Comment:
The updates you've made to the makefile, do they find the latest published version rather than including all the build info, eg commit and dirty

Reply:
Added support for when a user is on the latest version tag, it will find latest published image (or build it locally if user specifies build + run)